exchanges/order/order_test.go

```diff
@@ -2123,6 +2123,16 @@ func TestSideUnmarshal(t *testing.T) {
 	assert.ErrorAs(t, s.UnmarshalJSON([]byte(`14`)), &jErr, "non-string valid json is rejected")
 }
 
+func TestSideMarshal(t *testing.T) {
+	t.Parallel()
+	b, err := Buy.MarshalJSON()
+	assert.NoError(t, err, "Marshalling Side okay")
+	assert.Equal(t, `"BUY"`, string(b), "Correctly marshalled Side")
+	b, err = UnknownSide.MarshalJSON()
+	assert.NoError(t, err, "Marshalling Side okay")
+	assert.Equal(t, `"UNKNOWN"`, string(b), "Correctly marshalled Side")
```

Comment:
The text param for assertify is for when it fails, with the intent to highlight what is expected and is rendered to the output


I note that it's similar to the test above, but that can optionally be fixed too

Reply:
I rm'd them it was from auto complete